Gems/ScriptCanvas/Assets/TranslationAssets/Classes/MaterialData.names

```diff
@@ -525,13 +524,13 @@
                     "base": "GetBaseColor",
                     "context": "Getter",
                     "details": {
-                        "name": "Get Base Color"
```

Comment:
I see other methods have details name as "Get *", will this change cause trouble to those node name?

Reply:
The name is the visual/cosmetic part, the "base" and "context" are the parts that shouldn't be duplicate